examples/config/config

```diff
@@ -122,6 +122,26 @@ set completion_style  foreground="#eee8d5"
 set hint_style        weight="bold"
 
 set mode_section      <span background="#6c71c4" foreground="#eee8d5">[\@[\@mode_indicator]\@]</span>
+
+# As these indicators slow down uzbl, they're disabled by default
+set view_mode_indicator   \@- if ('\@page_view_mode' == 'web')   '<span foreground="#eee8d5" background="#586e75">'; else '<span foreground="#586e75" background="#eee8d5">'; -\@VM</span>
+# TODO find out why this fails
+#set css_indicator         \@- if (document.getElementsByTagName("link").length === 0)  '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#2aa198">'; -\@CS</span>
+set images_indicator      \@- if (\@autoload_images == 0)        '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#cb4b16">'; -\@IM</span>
+set cookie_indicator      \@- if ('\@cookie_policy' == 'never') '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#dc322f">'; -\@CO</span>
+set javascript_indicator  \@- if (\@enable_scripts == 0)         '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#d33682">'; -\@JS</span>
+set plugins_indicator     \@- if (\@enable_plugins == 0)         '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#b58900">'; -\@PL</span>
+set ssl_policy_indicator  \@- if ('\@ssl_policy' != 'fail')      '<span foreground="#eee8d5">'; else '<span foreground="#eee8d5" background="#268bd2">';-\@VS</span>
+#set indicator_section     <span foreground="cyan">@view_mode_indicator@css_indicator@images_indicator@cookie_indicator@javascript_indicator@plugins_indicator@ssl_policy_indicator</span>
+# NOTE the more of the indicators we include here (and thus actually display), the slower uzbl gets.
```

Comment:
Ok, I think I'm able to reproduce the bug now but lets take that over in that issue.

That problem aside it's likely not good to do this much work in the status format anyway because it's evaluated a lot. The approach taken by the scroll percentage is the way to go. i.e react to events and update simple variables that we later use in the format. ON_SET for most of them and then probably LOAD_FINISHED for the html/css one?